api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/fields/calculateSummaryValue.md

```diff
@@ -155,12 +195,22 @@ On the following image, pivot grid cells display indexes that show the order in
 
     <script type="text/javascript">
         function calculateSummaryValue (summaryCell) {
-            // Your code goes here
+            const prevCell = summaryCell.prev('column', true);
+            if (prevCell) {
+                const prevVal = prevCell.value("Total Amount");
+                const currentVal = summaryCell.value("Total Amount");
+                return currentVal - prevVal;
+            }
+            return null;
         }
     </script>
 
 ---
 
+The "Profit/Loss" cell is a difference between "Total Amount" cells in this case.
```

Comment:
I do not believe we can say that the "Profit/Loss" cell **is** a difference by itself. Could we say as follows instead?

```suggestion
The "Profit/Loss" cell displays a difference between "Total Amount" cells in this case.
```

OR

```suggestion
The "Profit/Loss" cell contains a difference between "Total Amount" cells in this case.
```

Other than this, this small merge request looks good to me, including pictures.